Controller.java

```diff
@@ -11,40 +11,29 @@
 @WebServlet("/hello")
 public class Controller extends HttpServlet {
 	
-	protected void Writers (HttpServletRequest request, HttpServletResponse response, String fname) 
+	private void sendResponse (HttpServletRequest request, String arg, HttpServletResponse response) 
 			throws ServletException, IOException {
 		try (PrintWriter writer = response.getWriter()){
-			fname = request.getParameter("fname");
-			writer.println("nick: " + fname);
+			writer.println(arg);
 	}
 }
 	protected void doGet(HttpServletRequest request, HttpServletResponse response) 
             throws ServletException, IOException {    
```

Comment:
Format you code here and everywhere, if you use Eclipse try use combination Ctrl+Shift+F. If you use Intellig IDEA try: Ctrl + Alt + L or Ctrl + Windows Key + Alt + L